clusterloader2/testing/watch-list/config.yaml

```diff
@@ -0,0 +1,74 @@
+# $testDuration controls
+# for how long the test should be executed
+# expressed in minutes
+{{$testDuration := 5}}
+name: huge-secrets
+namespace:
+  number: 1
+  prefix: "huge-secrets"
+tuningSets:
+- name: Uniform10qps
+  qpsLoad:
+    qps: 10
+steps:
+- name: Create secrets
+  phases:
+  - namespaceRange:
+      min: 1
+      max: 1
+      basename: huge-secrets
```

Comment:
I'm not sure if the namespaceRange is necessary when you set namespace.prefix on top.

Reply:
Also when I remove it I get the following error:
```
F0426 13:50:19.769252   46728 clusterloader.go:378] Test compilation failed: [config reading error: decoding failed: error unmarshaling JSON: while decoding JSON: json: cannot unmarshal object into Go struct field Step.steps.phases of type []*api.Phase]
exit status 1
```

Reply:
Sorry, my comment makes no sense.

I was thinking about namespaceRange.basename which seems to be redundant with namespace.prefix.